docs/tests/integration/test_agentd/test_agentd_reconnection.md

```diff
@@ -1,3 +1,44 @@
+# Test AgentD reconnection
```

Comment:
```suggestion
# Test wazuh-agentd reconnection
```

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/1235/commits/0dc8bbd8b5b38fa46f06e22dc11d5ea0a1d54cca